users/tests.py

```diff
@@ -78,3 +78,37 @@ def test_create_user_with_long_name(self):
                 last_name="last_name",
                 role=Role.EMPLOYEE,
                 team=TestTeams.valid_team)
+
+    @pytest.fixture
+    def example_team(self):
+        team = Team(name='a', description='b')
+        team.save()
+        return team
+
+    @pytest.fixture
+    def example_user_employee(self, example_team):
+        user = User.create_user('a', 'a@a.com', 'aaa', 'a', 'z', Role.MANAGER, example_team)
+        user.save()
+        return user
+
+    @pytest.fixture
+    def example_user_manager(self, example_team):
+        user = User.create_user('m', 'm@m.com', 'mmm', 'm', 'z', Role.MANAGER, example_team)
+        user.save()
+        return user
+
+    @pytest.mark.django_db
+    def test_check_is_employee_true(self, example_user_employee):
```

Comment:
maybe you can write one function for employees and one function for managers:
something like that:
```suggestion
    def test_check_is_employe(self, example_user_employee):
             if User.is_employee(example_user_employee)
                      return True
              return False
              
       and same for manager
    
```